draft-ietf-quic-transport.md

```diff
@@ -111,22 +114,26 @@ code and issues list for this draft can be found at
 
 # Overview
 
-QUIC is a multiplexed and secure general-purpose transport protocol that
-provides:
+QUIC is a secure general-purpose transport protocol. This
+document defines version 1 of QUIC, which conforms to the version-independent
+properties of QUIC defined in {{QUIC-INVARIANTS}}.
 
-* Stream multiplexing
+QUIC is a connection-oriented protocol that creates a stateful interaction
+between a client and server.
 
-* Stream- and connection-level flow control
-
-* Low-latency connection establishment
-
-* Connection migration and resilience to NAT rebinding
-
-* Authenticated and encrypted header and payload
-
-QUIC establishes a connection, which is a stateful interaction between a client
-and server. The primary purpose of a connection is to support the structured
-exchange of data by an application protocol.
+The QUIC handshake combines negotiation of cryptographic and transport
+parameters. QUIC integrates the TLS ({{?TLS13}}) handshake, although using a
+customized framing for protecting packets. The integration of TLS and QUIC is
+described in more detail in {{QUIC-TLS}}. The handshake is structured to permit
+the exchange of application data as soon as possible. This includes an option
+for clients to send data immediately (0-RTT), which might require prior
+communication to enable.
+
+Endpoints communicate in QUIC by exchanging QUIC packets. Most packets contain
+frames, which carry both control information and application data between
```

Comment:
To me, "both" makes implies that one frame carries both control and application data.

```suggestion
frames, which carry control information and application data between
```